docs/_guides/codewind-crw-quick-guide.md

```diff
@@ -0,0 +1,244 @@
+
+2\. Select **CodeReady Workspaces**. 
+
+3\. Select the **CodeReady Workspaces** cluster then **Create CheCluster**. 
+
+4\. Configure the fields, like **tlsSupport**, as needed. 
+
+5\. Copy and paste the following yaml into the provided text box: 
+
+```yaml
+apiVersion: org.eclipse.che/v1
+kind: CheCluster
+metadata:
+  name: codeready-workspaces
+spec:
+  server:
+    cheImageTag: ''
+    cheFlavor: codeready
+    devfileRegistryImage: ''
+    pluginRegistryImage: ''
+    tlsSupport: true
+    selfSignedCert: true
+    cheWorkspaceClusterRole: 'eclipse-codewind'
+    customCheProperties:
+      CHE_INFRA_KUBERNETES_WORKSPACE__START__TIMEOUT__MIN: "15"
+      CHE_LIMITS_WORKSPACE_IDLE_TIMEOUT: "0"
+      CHE_WORKSPACE_PLUGIN__BROKER_WAIT__TIMEOUT__MIN: "15"
+  database:
+    externalDb: false
+    chePostgresHostName: ''
+    chePostgresPort: ''
+    chePostgresUser: ''
+    chePostgresPassword: ''
+    chePostgresDb: ''
+  auth:
+    openShiftoAuth: false
+    identityProviderImage: ''
+    externalIdentityProvider: false
+    identityProviderURL: ''
+    identityProviderRealm: ''
+    identityProviderClientId: ''
+  storage:
+    pvcStrategy: per-workspace
+    pvcClaimSize: 1Gi
+    preCreateSubPaths: true
+```
+  * To manually install Codewind with CodeReady Workspaces, save this yaml to a disk and run `oc apply -f <yaml>`. 
+
+6\. CodeReady Workspaces now installs and you can access the CodeReady Workspaces URL and Red Hat SSO Admin Console URL:
+
+![Install CheCluster](./images/codereadyworkspaces-images/CRW-CheCluster.png){:width="900"}. 
+
+After you install CodeReady Workspaces, log in and create a Codewind workspace from the Codewind devfile: 
+
+1\. Log in to Che. Che loads. 
+
+2\. Go to **Workspaces** then click **Add Workspace**. 
+
+3\. Click **Import Devfile**. 
+
+4\. From **Source**, click **YAML**. 
+
+5\. Go to the link, [codewind-che-plugin/0.12.0/devfile.yaml](https://raw.githubusercontent.com/eclipse/codewind-che-plugin/0.12.0/devfiles/0.12.0/devfile.yaml), then copy and paste the contents into the YAML text box in your Codewind workspace. 
+
+6\. Click **Create & Open**.
+
+### Setting up Codewind
+Because of its dependency on `buildah`, Codewind needs to run as root and privileged. To enable Codewind, run the following commands: 
+  * `oc adm policy add-scc-to-user anyuid system:serviceaccounts:<namespace>:che-workspace` 
+  * `oc adm policy add-scc-to-user privileged system:serviceaccounts:<namespace>:che-workspace` 
+
+CodeReady Workspaces starts Codewind and installs the Codewind plug-ins. This process might take a couple of minutes for all of the necessary components to be pulled and started.
+
+### Configuring Codewind to use application stacks
+Configure Codewind to use Appsody templates so you can focus exclusively on your code. Complete the following steps to select the Appsody templates:
+
+1. Under the Explorer pane, select **Codewind**.
```

Comment:
These instructions are not quite right. It's still specific to Local Codewind and not remote.

1. On the right hand side, select the Codewind logo
2. Right click on `Projects`
3. Select **Template Source Manager**.
4. Enable **Appsody Stacks - incubator** and **Default templates**.